cloudformation/al_security_scanner_no_igw.template

```diff
@@ -106,66 +106,74 @@
         }
     },
     "Mappings": {
-        "AWSRegionAMI": {
-            "us-east-1": {
-                "US": "ami-0cfe1e2d48132cb1f",
-                "UK": "ami-0cfe1e2d48132cb1f"
+        "Internal": {
+            "Tags": {
+                "Version": "v1.0.18"
+            }
+        },
+        "RegionSettings": {
+            "af-south-1": {
+                "ImageId": "ami-0159f0cdf073852b7"
             },
-            "us-east-2": {
-                "US": "ami-0a0bfd0683191baa7",
-                "UK": "ami-0a0bfd0683191baa7"
+            "ap-east-1": {
+                "ImageId": "ami-00491099c15632859"
             },
-            "us-west-2": {
-                "US": "ami-05b6c462cc60392b9",
-                "UK": "ami-05b6c462cc60392b9"
+            "ap-northeast-1": {
+                "ImageId": "ami-00e5e1236a0190e66"
             },
```

Comment:
These element name changes are very different - this will for sure still work?

Reply:
hm, now that I think about it. it might not.